docs/source/data/data_catalog.md

```diff
@@ -337,7 +337,7 @@ The list of all available parameters is given in the [Paramiko documentation](ht
 
 ## Creating a Data Catalog YAML configuration file via CLI
 
-You can use [`kedro catalog create` command](../development/commands_reference.md#create-a-data-catalog-yaml-configuration-file) to create a Data Catalog YAML configuration.
+You can use [`kedro catalog create` command to create a Data Catalog YAML configuration](../development/commands_reference.md#create-a-data-catalog-yaml-configuration-file).
```

Comment:
You can use a [`kedro catalog create` command](../development/commands_reference.md#create-a-data-catalog-yaml-configuration-file) to create a Data Catalog YAML configuration.